### DISCUSSION TITLE
Training does not work with multiple videos!

### DISCUSSION THREAD

**Discussion by transkriptase** [talmolab/sleap#1764] [Help!]
Hello, 

i try to create a custom made good model to run more tracking on high computing system of my uni and/or on GPU computer, but I could not manage to do it even I try the basic commands. Is there a reason why? Because I know people who is working with multiple videos from differen species of same animal (mine are same species and same setup). the command I  run is simple sleap-train baseline.centrod.json (I also tried bottomup) --video_paths "video1" "video2"......video25"

And the error I get is 
"TypeError: '>' not supported between instances of 'NoneType' and 'int'" 

Can you please help me!
Best,

**Comment by eberrigan** [comment]
Hi @transkriptase,

Please see our CLI reference here: [https://sleap.ai/develop/guides/cli.html](https://sleap.ai/develop/guides/cli.html). 

Can you confirm SLEAP is installed first with `sleap-train --help`.

- I am not sure where your files are located relative to your current directory, but getting all of the video-paths correct can be a pain. You can export a sleap-labels package in the GUI on your desktop computer and then train using that on your HPC. 
- Please try a comma delimiter between video-paths [CLI definition](https://github.com/talmolab/sleap/blob/0134ab443663bf1b1f0287d632dce9fdef80c14b/sleap/nn/training.py#L1938)
-  `--video-paths` is only used when the videos in your labels file are not accessible.
-  Is your `labels_path` specified in your training config file?

Please copy and paste your exact command and error to help troubleshoot the error. 

Thanks,

Elizabeth
